main_page.py

```diff
@@ -4,89 +4,104 @@
+    data = util.safe_to_pandas(conn, query)
+    return data
 
-data = cs.fetch_pandas_all()
-data['YEAR_MONTH'] = pd.to_datetime(data['YEAR_MONTH'], format='%Y-%m').dt.date
+@st.cache_data
+def age_data():
+    query = """SELECT CASE 
+                        WHEN div0(current_date() - BIRTH_DATE, 365) < 49 THEN '34-48'
+                        WHEN div0(current_date() - BIRTH_DATE, 365) >= 49 AND div0(current_date() - BIRTH_DATE, 365) < 65 THEN '49-64'
+                        WHEN div0(current_date() - BIRTH_DATE, 365) >= 65 AND div0(current_date() - BIRTH_DATE, 365) < 79 THEN '65-78'
+                        WHEN div0(current_date() - BIRTH_DATE, 365) >= 79 AND div0(current_date() - BIRTH_DATE, 365) < 99 THEN '79-98'
+                        WHEN div0(current_date() - BIRTH_DATE, 365) >= 99 THEN '99+' END
+                AS AGE_GROUP,
+                COUNT(*) AS COUNT
+                FROM TUVA_PROJECT_DEMO.CORE.PATIENT
+                GROUP BY 1
+                ORDER BY 1;"""
+    data = util.safe_to_pandas(conn, query)
+    return data
+data = summary_data()
+demo_gender = gender_data()
+demo_race = race_data()
+demo_age = age_data()
 
-st.markdown("### Summary of Claims")
+st.markdown("# Summary of Claims")
 start_date, end_date = st.select_slider("Select date range for claims summary",
-                                        options=data['YEAR_MONTH'].sort_values(),
-                                        value=(data['YEAR_MONTH'].min(), data['YEAR_MONTH'].max()))
-filtered_data = data.loc[(data['YEAR_MONTH'] >= start_date) & (data['YEAR_MONTH'] <= end_date), :]
+                                        options=data['year_month'].sort_values(),
+                                        value=(data['year_month'].min(), data['year_month'].max()))
+filtered_data = data.loc[(data['year_month'] >= start_date) & (data['year_month'] <= end_date), :]
+
+st.markdown("### High Level Summary")
+st.markdown("""At a glance, see the total medical spend and PMPM for the chosen time period. As well as a trend
+graph for other important financial metrics""")
+st.sidebar.markdown("# Claims Summary")
 
 # Summary Metrics
-total_spend = filtered_data['MEDICAL_SPEND'].sum()
-total_member_months = filtered_data['MEMBER_MONTH_COUNT'].sum()
+total_spend = filtered_data['medical_spend'].sum()
+total_member_months = filtered_data['member_month_count'].sum()
 avg_pmpm = total_spend/total_member_months
 
-col1, col2, col3 = st.columns([1.5,1,1])
-col1.metric("Total Medical Spend", '${:,.2f}'.format(total_spend))
-col2.metric("Total Member Months", total_member_months)
-col3.metric("Average PMPM", '${:,.2f}'.format(avg_pmpm))
+col1, col2, col3 = st.columns([1,1,1])
+col1.metric("Total Medical Spend", '${}'.format(util.human_format(total_spend)))
+col2.metric("Total Member Months", util.human_format(total_member_months))
+col3.metric("Average PMPM", '${}'.format(util.human_format(avg_pmpm)))
```

Comment:
This is awesome -- I'll copy this into the financial breakdown. Should we add pharmacy spend?